ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -328,7 +328,9 @@ impl<'a> TraceBuilder<'a> {
                 }
                 None => {
                     // UnmappableBlock block
-                    todo!();
```

Comment:
Why have we removed this? I know we want to do one day, but we can't do so yet can we?

Reply:
No real test case will be testable if we crash out here.

Reply:
Wouldn't it be better to leave it in until/unless that does happen?

Reply:
If we don't ignore unmappable traces, we can't continue dealing with the current sample trace we got. In the future we would have to put outlining logic here probably.

Reply:
OK.